locales/en/email-strings.ftl

```diff
@@ -82,11 +82,14 @@ email-verify-subhead = Verify your email to start protecting your data after a b
 email-verify-simply-click = Simply click the link below to finish verifying your account.
 
 ## Breach report
-## Variables:
-##   $email-address (string) - Email address
 
 email-breach-summary = Here’s your data breach summary
+# Variables:
+#   $email-address (string) - Email address, bolded
```

Comment:
Nit: The “bolded” string is the `email-breach-detected-2` one below, right?

Reply:
The original one included the `<b>` inside the variable (which Fluent-React doesn't allow us to do), whereas the new one gives the localiser more freedom to bold the relevant text (and works with Fluent-React).